packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3944,4 +3944,69 @@ void main() {
     await tester.pump();
     expect(focusNode3.hasPrimaryFocus, isTrue);
   });
+
+  testWidgets('Cupertino text field semantics', (WidgetTester tester) async {
+    final SemanticsHandle handle = tester.ensureSemantics();
```

Comment:
I don't think you need to create and dispose this handle. I'm pretty sure semantics are enabled by default in all widget tests.

https://api.flutter.dev/flutter/flutter_test/testWidgets.html

Reply:
you're right, done